includes/wikia/services/UserStatsService.class.php

```diff
@@ -134,30 +131,44 @@ function () use ( $flags ) {
 					$stats['lastContributionTimestamp'] = $this->initLastContributionTimestamp();
 				}
 
+				if ( $stats->needsUpdate() ) {
+					$this->scheduleStatsUpdateTask( $stats );
+				}
+
 				return $stats;
 			}
 		);
 		return $stats;
 	}
 
+	/**
+	 * Reset edit count of this user - force a recalculation and trigger update
+	 */
+	public function resetEditCount() {
+		$stats = $this->getStats();
```

Comment:
`getStats()` schedules a task when needed, and here the same task is scheduled again?